src/main/java/co/omise/Example.java

```diff
@@ -334,27 +334,30 @@ void retrieveRefund() throws IOException, OmiseException, ClientException {
     }
 
     void createToken() throws IOException, OmiseException, ClientException {
-        Token token = client().tokens()
-                .create(new Token.Create().card(new Card.Create()
+        Request<Token> request = new Token.CreateRequestBuilder()
+                .card(new Card.Create()
                         .name("Somchai Prasert")
                         .number("4242424242424242")
                         .expirationMonth(10)
-                        .expirationYear(2018)
+                        .expirationYear(2022)
                         .city("Bangkok")
                         .postalCode("10320")
                         .securityCode("123"))
-                );
+                .build();
+        Token token = client().sendRequest(request, Token.class);
         System.out.printf("created token: %s", token.getId());
     }
 
     void retrieveToken() throws IOException, OmiseException, ClientException {
-        Token token = client().tokens().get("tokn_test_4xs9408a642a1htto8z");
+        Request<Token> request = new Token.GetRequestBuilder("tokn_test_4xs9408a642a1htto8z").build();
+        Token token = client().sendRequest(request, Token.class);
         System.out.printf("token last digits: %s", token.getCard().getLastDigits());
     }
 
     void listTransactions() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Transaction>> request = new Transaction.ListRequestBuilder().build();
-        ScopedList<Transaction> transactions = client().sendRequest(request, new TypeReference<ScopedList<Transaction>>() {});
+        ScopedList<Transaction> transactions = client().sendRequest(request, new TypeReference<ScopedList<Transaction>>() {
+        });
```

Comment:
Can we remove the newline here?

Reply:
We can, but it formats it this every time I autoformat the code. It kind of bothers me as well. Fortunately it doesn't behave the same way in `Kotlin`